signifyd/signifyd.module

```diff
@@ -54,4 +54,58 @@ class SignifydApi {
 
     return $resp;
   }
+  
+  /**
+	* Get a case details from Signifyd.
+	*/
+	static function get_case($id) {
+		$api_key  = variable_get('signifyd_api_key', '');
+		$api_base = variable_get('signifyd_api_url', '');
+		$api_url  = $api_base . '/cases/'.$id;
+		
+		$resp = drupal_http_request($api_url, array(
+			'method' => 'GET',
+			'headers' => array(
+				'Content-Type' => 'application/json',
+				'Authorization' => 'Basic ' . base64_encode($api_key),
+			),
+		 ));
+		
+		watchdog('signifyd', 'API Get Response: <pre>'.print_r($resp, true).'</pre>');
+		
+		return $resp;
+	}
+	
+	/**
+	* Get a guarentee canceled and refunded
+	*/
+	static function cancel_guarentee($id) {
+		$api_key  = variable_get('signifyd_api_key', '');
+		$api_base = variable_get('signifyd_api_url', '');
+		$api_url  = $api_base . '/cases/'.$id.'/guarantee';
+		
+		$ch = curl_init();
```

Comment:
is it not possible to use the drupal_http client instead of curl?

Reply:
Had issues with it... Trying to remember what the exact issues were... I ended up just using curl because I knew with certainty it would work. I'll go back in there and see.

Reply:
The latest commit has the requested changes! Glad you called that out.